apps/coscrad-frontend/src/components/resource-info/presenters/resource-info.presenter.tsx

```diff
@@ -14,7 +15,7 @@ export const ResourceInfoPresenter = ({
  */
 IAggregateInfo<ResourceType>): JSX.Element => (
     <Link to={`/${routes.resources.ofType(resourceType).index}`}>
-        <Card>
+        <PrimaryCard>
```

Comment:
We should probably emphasize in the name that this is just styling that we are wrapping in.

Reply:
Yes, I like the idea of a convention so we're always clear it's just styling